src/osd/ECBackend.cc

```diff
@@ -1011,9 +1011,11 @@ void ECBackend::handle_sub_read(
 	j->get<1>(),
 	bl, j->get<2>());
       if (r < 0) {
-	get_parent()->clog_error() << "Error " << r
-				   << " reading object "
-				   << i->first;
+        if (r != -ENOENT) {
```

Comment:
@dillaman @gregsfortytwo I don't think we want to skip ENOENT logging here.

See #16663.  My pull request makes it possible for cmpext to trust ENOENT.